Update of .gitignore

yorugac opened the pull request:
Closes #2

yorugac committed:
Travis CI: fix Android SDK tools

yorugac committed:
update .gitignore

Commit:
Merge branch 'master' into master

jo-elimu commented on .travis.yml:
```diff
@@ -6,6 +6,7 @@ language: android
 android:
   components:
+    - tools
     - build-tools-23.0.3
```
@yorugac Try changing from 23.0.3 to 23.0.1

yorugac committed:
change of build tools; Travis CI: compiler update

yorugac committed:
Travis CI: fix dependencies

yorugac committed:
Firebase is removed from the game.

The following functionality was deleted:
- ads, reward videos
- invites
- messages
- analytics
- remote configuration of menu
